release_notes/48-release-notes.adoc

```diff
@@ -419,4 +420,24 @@ This release also addresses the following security vulnerabilities:
 * Flaw in libxml2 library (link:https://access.redhat.com/security/cve/CVE-2025-32415[CVE-2025-32415])
 * Perl standard library threads component flaw (link:https://access.redhat.com/security/cve/CVE-2025-40909[CVE-2025-40909])
 
+
+[id="about-this-release-483_{context}"]
+== About release 4.8.3
```

Comment:
Please add a "technical change" kind of note saying something along these lines

> Starting from version 4.8.3, product container images are built and released using a different set of technologies as compared to earlier versions. This may bring changes to metadata such as container labels, contents and location of SBOMs, and so on. These changes do not affect the functioning of the product but may impact third-party integrations.

https://github.com/stackrox/stackrox/blob/b16884ab7ff0fc5956ad3a3211e789aaf3b103b1/CHANGELOG.md?plain=1#L18